src/modele/Technician.java

```diff
@@ -154,4 +184,40 @@ public boolean addPotentialInstallation(Machine machine) {
         }
         return false;
     }
+
+    /**
+     * Checks if the technician can install the machines requested and if he can
+     * work (doesn't need to rest)
+     *
+     * @todo : 2 days rest if 5 working days, else 1 day rest
+     * @todo : to review
+     * @param demand
+     * @param dayNumber
+     * @return true if success
+     */
+    public boolean canInstallDemand(Demand demand, int dayNumber) {
+        boolean canInstall = false;
+        for (Installation installation : this.potentialInstallations) {
+            if (installation.getMachine().equals(demand.getMachine())) {
+                canInstall = true;
```

Comment:
if (!installation.getMachine().equals(demand.getMachine())) {
                return false

?

Reply:
to check whether the technician can install the machine
I agree Installation should be called Accreditation and potentialInstallations accreditations